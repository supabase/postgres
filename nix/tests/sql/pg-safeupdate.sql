load 'safeupdate';

set safeupdate.enabled=1;

create schema v;

create table v.foo(
  id int,
  val text
);

update v.foo
  set val = 'bar';

drop schema v cascade;
