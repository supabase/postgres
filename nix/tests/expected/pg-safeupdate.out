load 'safeupdate';
set safeupdate.enabled=1;
create schema v;
create table v.foo(
  id int,
  val text
);
update v.foo
  set val = 'bar';
ERROR:  UPDATE requires a WHERE clause
drop schema v cascade;
NOTICE:  drop cascades to table v.foo
